src/pt/randomscan/src/eu/kanade/tachiyomi/extension/pt/randomscan/LuraToon.kt

```diff
@@ -113,28 +155,34 @@ class LuraToon : HttpSource(), ConfigurableSource {
             }
     }
 
+    override fun chapterListParse(response: Response) = throw UnsupportedOperationException()
+
     fun chapterListParse(manga: SManga, response: Response): List<SChapter> {
         if (response.code == 404) {
             throw Exception("Capitulos não encontrados, tente migrar o manga, alguns nomes da LuraToon mudaram")
         }
 
-        val comics = response.parseAs<Manga>()
+        val comics = response.parseAs<MangaDTO>()
 
         return comics.caps.sortedByDescending {
             it.num
         }.map { chapterFromElement(manga, it) }
     }
 
-    private fun chapterFromElement(manga: SManga, capitulo: Capitulo) = SChapter.create().apply {
+    private fun chapterFromElement(manga: SManga, capitulo: CapituloDTO) = SChapter.create().apply {
         val capSlug = capitulo.slug.trimStart('/')
         val mangaUrl = manga.url.trimEnd('/').trimStart('/')
-        setUrlWithoutDomain("/api/obra/$mangaUrl/$capSlug")
+        setUrlWithoutDomain("/api/484d2a13/$mangaUrl/$capSlug") // api/484d2a13/slug_obra/slug
```

Comment:
```suggestion
        url = "$mangaUrl/$capSlug"
```

Then use `pageListRequest` to create the api call

Reply:
I've already updated it in my debug code, and I'll include it in the next PR along with the pagination solution. Thanks.